src/filesystem.cpp

```diff
@@ -1698,7 +1716,7 @@ std::string get_program_invocation(const std::string& program_name)
 #endif
 	);
 
-	return (bfs::path(game_config::wesnoth_program_dir) / real_program_name).string();
```

Comment:
`game_config::wesnoth_program_dir` can be completely removed now. It's not used anywhere else.

Reply:
Updated.